File: ordenarArquivos/src/main/java/ordenarArquivos/model/Capa.java
package ordenarArquivos.model;

public class Capa {

	private String arquivo;
	private TipoCapa tipo;

	public String getArquivo() {
		return arquivo;
	}

	public void setArquivo(String arquivo) {
		this.arquivo = arquivo;
	}

	public TipoCapa getTipo() {
		return tipo;
	}

	public void setTipo(TipoCapa tipo) {
		this.tipo = tipo;
	}

	public Capa(String arquivo, TipoCapa tipo) {
		this.arquivo = arquivo;
		this.tipo = tipo;
	}

	public Capa() {
		this.arquivo = "";
		this.tipo = TipoCapa.CAPA;
	}

}

File: OrdenarArquivos/src/main/kotlin/com/fenix/ordenararquivos/model/entities/Capa.kt
package com.fenix.ordenararquivos.model.entities

import com.fenix.ordenararquivos.model.enums.TipoCapa

data class Capa(
    var nome: String = "",
    var arquivo: String = "",
    var tipo: TipoCapa = TipoCapa.CAPA,
    var isDupla: Boolean = false,
    var direita: Capa? = null
) { }